internal/server/edit_payment.go

```diff
@@ -28,7 +28,7 @@ type editPaymentData struct {
 
 func EditPayment(client EditPaymentClient, tmpl template.Template) Handler {
 	return func(w http.ResponseWriter, r *http.Request) error {
-		caseID, err := strconv.Atoi(r.FormValue("id"))
+		_, err := strconv.Atoi(r.FormValue("id"))
```

Comment:
I've left this in to validate the url parameter although the parameter is no longer used

Reply:
I dont think this shouldn't be needed anymore, we should only expect the payment ID url param now :)